test/jasmine/tests/click_test.js

```diff
@@ -809,3 +810,48 @@ describe('Test click interactions:', function() {
         });
     });
 });
+
+describe('dragbox', function() {
+
+    afterEach(destroyGraphDiv);
+
+    it('should scale subplot and inverse scale scatter points', function(done) {
+        var mock = Lib.extendDeep({}, require('@mocks/bar_line.json'));
+
+        function assertScale(node, x, y) {
+            var scale = Drawing.getScale(node);
+            expect(scale.x).toBeCloseTo(x, 1);
+            expect(scale.y).toBeCloseTo(y, 1);
+        }
+
+        Plotly.plot(createGraphDiv(), mock).then(function() {
+            var node = d3.select('rect.nedrag').node();
+            var pos = getRectCenter(node);
+
+            assertScale(d3.select('.plot').node(), 1, 1);
+
+            d3.selectAll('.point').each(function() {
+                assertScale(this, 1, 1);
+            });
+
+            mouseEvent('mousemove', pos[0], pos[1]);
+            mouseEvent('mousedown', pos[0], pos[1]);
+            mouseEvent('mousemove', pos[0] + 50, pos[1]);
+
+            setTimeout(function() {
+                assertScale(d3.select('.plot').node(), 1.14, 1);
+
+                d3.select('.scatterlayer').selectAll('.point').each(function() {
+                    assertScale(this, 0.87, 1);
+                });
+                d3.select('.barlayer').selectAll('.point').each(function() {
+                    assertScale(this, 1, 1);
+                });
+
+                mouseEvent('mouseup', pos[0] + 50, pos[1]);
```

Comment:
Fun fact. This test case doesn't need a `mouseup` event to test what is desired. But, we need to trigger one to not affect other tests downstream (namely in `config_test.js`).